packages/sources/paxos/src/endpoint/assetAttestations.ts

```diff
@@ -26,7 +26,7 @@ const getAttestationURI = (asset: string) => `/asset-attestations/${asset.toUppe
 
 const toAttestation = async (config: Config, asset: string): Promise<Attestation> => {
   const url = getAttestationURI(asset)
-  const reqConfig = { ...config.api, baseURL: BASE_URL, url }
+  const reqConfig = { ...config.api, baseURL: DEFAULT_BASE_URL, url }
```

Comment:
This should be built up in `config.ts` and brought in with the spread of `config.api`.